### PR TITLE
Added implementation of new webhook type

### LINKED ISSUE #13: Add new webhook class
### Motivation

Why is this feature required? What problems does it solve?
Until last week, we had only webhook, which now is legacy webhook. This webhook used the class DeliveryWebhookModel. We don't have a class for the new webhook which is using different naming for properties.

### Proposed solution

An ideal solution for the above problems would be to create a class for the new webhook type so all the projects that use this webhook won't be required to create a new class in each project.

### PR DESCRIPTION
**Pull request by EduardDumitru**
### Motivation

Which issue does this fix? Fixes [#`issue number`](https://github.com/kontent-ai/aspnetcore-extensions/issues/13)

### Checklist

- [x] Code follows coding conventions held in this repo
- [ ] Automated tests have been added
- [ ] Tests are passing
- [ ] Docs have been updated (if applicable)
- [ ] Temporary settings (e.g. variables used during development and testing) have been reverted to defaults

### CODE REVIEW

**Comment by pokornyd** [comment]
Hello Eduard, thank you for your PR submission. I'll review the proposed changes and get back to you with updates.

File: Kontent.Ai.AspNetCore/Webhooks/Models/WebhookNotification.cs [hunk 1]
```diff
@@ -0,0 +1,138 @@
+﻿using System;
+using System.Text.Json.Serialization;
+
+namespace Kontent.Ai.AspNetCore.Webhooks.Models
+{
+    /// <summary>
+    /// Root object of a Kontent.ai triggered webhook.
+    /// See <see href="https://kontent.ai/learn/docs/webhooks/webhooks/net">webhooks reference documentation</see> for details.
+    /// </summary>
+    public class WebhookNotification
```
Comment (IvanKiral) [inline Kontent.Ai.AspNetCore/Webhooks/Models/WebhookNotification.cs:10]: Please update the Readme to reflect the changes

File: Kontent.Ai.AspNetCore/Webhooks/Models/WebhookNotification.cs [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+        /// <summary>
+        /// The object's ID.
+        /// </summary>
+        [JsonPropertyName("id"), JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
+        public Guid? Id { get; set; }
+
+        /// <summary>
+        /// The item's name.
+        /// </summary>
+        [JsonPropertyName("name"), JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
```
Comment (IvanKiral) [inline Kontent.Ai.AspNetCore/Webhooks/Models/WebhookNotification.cs:93]: are those `JsonIgnoreCondition` correct? 

from documentation
WhenWritingNull - The property is ignored on serialization if it's a reference type null, or a nullable value type null.

this property is not even nullable or am I missing something?

https://learn.microsoft.com/en-us/dotnet/standard/serialization/system-text-json/ignore-properties

File: Kontent.Ai.AspNetCore/Webhooks/Models/WebhookNotification.cs (hunk 1 above)
Comment (IvanKiral) [inline Kontent.Ai.AspNetCore/Webhooks/Models/WebhookNotification.cs:10]: Moreover, I would add snapshot test for this model :D

**Review by IvanKiral — CHANGES_REQUESTED** [review]

File: Kontent.Ai.AspNetCore/Webhooks/Models/WebhookNotification.cs (hunk 2 above)
Comment (pokornyd) [reply Kontent.Ai.AspNetCore/Webhooks/Models/WebhookNotification.cs:93]: reference types are always nullable, `?` added where applicable. I'm not 100% sure about the decorators intended purpose but I followed the same approach as was used previously.

File: Kontent.Ai.AspNetCore/Webhooks/Models/WebhookNotification.cs (hunk 1 above)
Comment (pokornyd) [reply Kontent.Ai.AspNetCore/Webhooks/Models/WebhookNotification.cs:10]: added a deserialization test

**Review by IvanKiral — APPROVED** [review]

**Review by pokornyd — APPROVED** [review]

**Pull request merged**